src/features/presence/Presence.js

```diff
@@ -159,6 +160,11 @@ class Presence extends Component {
 
     _renderTimestampMessage = (interactedAt, interactionType, isActive) => {
         const lastActionMessage = determineInteractionMessage(interactionType);
+        const { intl } = this.props;
+        const timeAgo = intl.formatRelativeTime
```

Comment:
Uses new version if available, otherwise old version.
Need to double check if the output is correct, specially the delta.